server/game/cards/11-TCaR/Mausoleum.js

```diff
@@ -0,0 +1,22 @@
+const DeedCard = require('../../deedcard.js');
+
+class Mausoleum extends DeedCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'Mausoleum',
+            cost: ability.costs.bootSelf(),
+            when: {
+                onCardAced: event =>
+                    !event.card.hasOneOfKeywords(['token', 'abomination']) && event.card.getType() === 'dude' && EventTarget.originalLocation === 'play area'
```

Comment:
Probably a typo or incorrect code suggestion.
```suggestion
                    !event.card.hasOneOfKeywords(['token', 'abomination']) && event.card.getType() === 'dude' && event.originalLocation === 'play area'
```